src/scene/shader-lib/chunks/standard/frag/clearCoatGloss.js

```diff
@@ -18,6 +18,10 @@ void getClearCoatGlossiness() {
     ccGlossiness *= saturate(vVertexColor.$VC);
     #endif
 
+    #ifdef MAPINVERT
+    ccGlossiness = 1.0 - ccGlossiness;
```

Comment:
I brought this up in a discussion previously and was told that roughness is definitely **not** 1 - gloss. What's the deal here? cc @GSterbrant @mvaligursky

Reply:
This is not my invention. It was transferred from our previous set of custom chunks in glb-parser.

Reply:
Fair enough. I did just do a bit of Googling and there are several posts saying roughness is just gloss inverted. But anyway, seems this isn't a major concern... 😄